### PR TITLE
Sticky panel

### PR DESCRIPTION
**Pull request by lal65**
See demo at https://developers.outreach.psu.edu/sticky-panel/?p=viewall-organisms-sticky-panel

### CODE REVIEW

File: packages/sticky-panel/src/scss/styles.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
+@import "../../../base/src/scss/imports/variables";
+
+.sticky-panel {
+  position: sticky;
+  top: 0;
+
+  .sticky-panel__content {
```
Comment (kjl16) [inline packages/sticky-panel/src/scss/styles.scss:7]: Can simplify with &__content

**Review by kjl16 — CHANGES_REQUESTED** [review]
One minor thing.

File: packages/sticky-panel/src/scss/styles.scss (hunk 1 above)
Comment (lal65) [reply packages/sticky-panel/src/scss/styles.scss]: @kjl16 - thanks.  Applied the simplification.

**Review by kjl16 — APPROVED** [review]
Looks good - ship it.

**Pull request merged**